docs/mkdocs.yml

```diff
@@ -34,33 +34,56 @@ site_dir: 'generated'
 nav:
   - Welcome:
       - Landing: index.md 
-  - Overview:
-      - Readme: readme.md
-      - User guide: direct/user-guide.md
+  - What is KubeStellar?:
+      - Overview: readme.md
+#      - User guide: direct/user-guide.md
       - Architecture: direct/architecture.md
+      - Related:
+        - KubeFlex: direct/kubeflex-intro.md
+        - KubeStellar Galaxy: direct/galaxy-intro.md
       - Release-notes: direct/release-notes.md
-  - Getting Started: 
-          - Pre-reqs: direct/pre-reqs.md
+  - Using Kubestellar:
+          - Basics: direct/usage-basics.md
+          - Prerequisites: direct/pre-reqs.md
+#              - Pre-reqs: direct/pre-reqs.md
           - Setting up KubeStellar:
-              - Using existing hosting cluster: direct/hosting-cluster.md
-              - KubeStellar on KIND: direct/ks-on-kind.md
-              - KubeStellar on K3D: direct/deploy-on-k3d.md
-              - KubeStellar on OCP: direct/ks-on-ocp.md
+              - Set Up Kubestellar Core Components: # common-subs/placeholder.md
+                - Try the "Common-setup": direct/common-setup-intro.md
+                - Step by Step: direct/common-setup-step-by-step.md
+                - (experimental) Via Helm Chart: direct/core-chart.md
+#                - Prepare the environment: common-subs/placeholder.md
+#                - Initialize Kubeflex: common-subs/placeholder.md
+#                - Install Kubestellar Core components: common-subs/placeholder.md
+#                - Create the Inventory & Transport Space: common-subs/placeholder.md
+#                - Create the Workload Description Space: common-subs/placeholder.md
+              - Set up and Register Workload Execution Clusters (WECs): direct/example-wecs.md
+#              - Register WECS with core: common-subs/placeholder.md
+              - Commission Workloads: direct/commisssion-workloads.md
+#          - Modifying Workloads: common-subs/placeholder.md
+          - Installation & Usage Examples:
+              - Example Cluster Preparation: direct/hosting-cluster.md
```

Comment:
This particular file is specifically about the usage of a _pre-existing_ hosting cluster, in contrast the other content that _creates_ the hosting cluster.